tools/Google.Cloud.Tools.ReleaseManager/AddCommand.cs

```diff
@@ -46,6 +54,22 @@ protected override int ExecuteImpl(string[] args)
             }
             var root = DirectoryLayout.DetermineRootDirectory();
             var googleapis = Path.Combine(root, "googleapis");
+
+            if (pull)
+            {
+                Console.WriteLine($"Pulling googleapis directory");
+                using var repo = new Repository(googleapis);
+                string oldSha = repo.Head.Tip.Sha;
+                var merger = new Signature("Ignored", "ignored@google.com", DateTimeOffset.UtcNow);
+                var options = new PullOptions
+                {
+                    MergeOptions = new MergeOptions { FastForwardStrategy = FastForwardStrategy.FastForwardOnly, FailOnConflict = true }
+                };
+                Commands.Pull(repo, merger, options: null);
```

Comment:
Created `PullOptions` not used in the `Pull` call.

Reply:
Doh, thanks. Interesting that it works anyway, but I didn't validate that it fails if there are conflicts etc. Will use the options.